package.json

```diff
@@ -55,14 +52,9 @@
   "devDependencies": {
     "ava": "0.22.0",
     "fs-extra": "4.0.1",
-    "pre-commit": "1.2.2"
-  },
-  "ava": {
-    "files": [
-      "test/**/*.js"
-    ],
-    "source": [
-      "lib/**/*.js"
-    ]
+    "ts-loader": "^4.2.0",
```

Comment:
Are we actually using these 4 dependencies? I thought we took out webpack and typescript.

Reply:
Good catch 👍